features/270-next-skycli/commands.md

```diff
@@ -42,7 +42,7 @@ app=myapp
 
 `skycli config set-app [APP_NAME]`
 
-Change skycli current app context to `APP_NAME`
+Change skycli current app context to `APP_NAME` by creating or updating `./skygear.yaml`.
```

Comment:
Creating? When it creates how it knows which "parent" directory is the right one?

Reply:
ummm... Actually, there are 2 cases that user need to set app. 
1. Using general app commands (add user, update tenant config...etc.) without CF
2. Deploy CF

We don't need to worry about case 2 as user must need to write the config to deploy CF. And the problem is case 1. 

In order to avoid confusion and let user know skycli use app's directory config file to config app, is it better to remove this command? But user will need to write the minimal config file for case 1.  Thought?

Reply:
What about have a command for user to init a new app directory? Like create app etc?

For general app commands, they must do it in the app directory or else we will tell user there are no app in the current directory?

Just imagining the use cases, not sure if it is the easiest to use / understand

Reply:
> For general app commands, they must do it in the app directory or else we will tell user there are no app in the current directory?

Yes, I assume validation is needed. If user call commands without login or config file, we should let user know.

> What about have a command for user to init a new app directory? Like create app etc?

If this is `init` command, i think it should help init the project directory too. I updated the PR to remove `config set-app`, update `app create` and add `app init` commands.